data/npc/lib/npc.lua

```diff
@@ -11,33 +11,34 @@ function msgcontains(message, keyword)
 end
 
 function doNpcSellItem(cid, itemid, amount, subType, ignoreCap, inBackpacks, backpack)
+	local player = Player(cid)
 	local amount = amount or 1
 	local subType = subType or 0
 	local item = 0
-	if isItemStackable(itemid) then
+	if ItemType(itemid):isStackable() then
 		if inBackpacks then
-			stuff = doCreateItemEx(backpack, 1)
-			item = doAddContainerItem(stuff, itemid, math.min(100, amount))
+			stuff = Game.createItem(backpack, 1)
+			item = doAddContainerItem(stuff, itemid, math.min(100, amount)) -- ???
```

Comment:
Why did you add "-- ???"
And why did you not change the compat function? https://github.com/otland/forgottenserver/blob/master/data/lib/compat/compat.lua#L583-L594